### DISCUSSION TITLE
Are the measurements for primary cuts inclusive of trimming? Or applied after trimming?

### DISCUSSION THREAD

**Discussion by contentfree** [lairdubois/lairdubois-opencutlist-sketchup-extension#464] [Q&A] [answered]
I have the default ½" trimming set for a standard 4'x8' sheet of plywood. The first primary cut is given from the edge. My question: Does that measurement from the edge include the trimming? Or is the expectation that all trimming will be done first?

**Comment by bbeaulant** [comment, marked as answer]
Yes it does.

<img width="551" alt="Capture d’écran 2023-01-23 à 08 20 08" src="https://user-images.githubusercontent.com/156056/213984991-9c3344f3-93bb-4e45-9dc5-33d5beba8650.png">
